python/lsst/pipe/tasks/subtractBrightStars.py

```diff
@@ -124,7 +152,23 @@ class SubtractBrightStarsConfig(PipelineTaskConfig, pipelineConnections=Subtract
         doc="Magnitude limit, in Gaia G; all stars brighter than this value will be subtracted",
         default=18,
     )
-    warpingKernelName = ChoiceField[str](
+    minValidAnnulusFraction = Field[float](
+        dtype=float,
+        doc="Minimum number of valid pixels that must fall within the annulus for the bright star to be "
+        "saved for subsequent generation of a PSF.",
+        default=0.0,
+    )
+    numSigmaClip = Field[float](
+        dtype=float,
+        doc="Sigma for outlier rejection; ignored if annularFluxStatistic != 'MEANCLIP'.",
+        default=4,
+    )
+    numIter = Field[int](
+        dtype=int,
```

Comment:
remove redundant dtype

Reply:
Redundant `dtype` args removed here and elsewhere where appropriate. Typing arguments supplied where previously missing.